WordPress/Classes/Services/PostService.h

```diff
@@ -1,128 +1,7 @@
-#import <Foundation/Foundation.h>
-#import "LocalCoreDataService.h"
-#import "PostServiceOptions.h"
 
-@class Blog, Post, Page, AbstractPost;
-@class RemotePost;
-
-typedef NS_ENUM(NSInteger, PostServiceType) {
-    PostServiceTypePost,
-    PostServiceTypePage,
-    PostServiceTypeAny
-};
+typedef NSString * PostServiceType;
+extern NSString * const PostServiceTypePost;
+extern NSString * const PostServiceTypePage;
+extern NSString * const PostServiceTypeAny;
```

Comment:
Ah, how about a helper method in `PostService` that exports the string value? Or instead convert `PostService.h` to something like `PostServiceTypes.h` and then import into Swift?

Just want to avoid having both two "PostServices" and maintaining duplicate string values.